packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -4,7 +4,7 @@
 section: extensions
```

Comment:
You could update this section to `AI-infra-ui-components` since i don't forsee us moving these components to patternfly.org particularly soon. 

You might also need to update:
[This line](https://github.com/patternfly/AI-infra-ui-components/blob/main/packages/module/patternfly-docs/patternfly-docs.config.js#L3) so that `extensions` is replaces with AI infra ui components
[This line](https://github.com/patternfly/AI-infra-ui-components/blob/main/packages/module/patternfly-docs/patternfly-docs.source.js#L11) in the same way.

Reply:
Thanks for referencing the other lines, I tried updating this before, but didn't realize I have to change the section name in other places too